addons/mass_mailing/models/mailing.py

```diff
@@ -744,7 +744,9 @@ def _get_seen_list(self):
             SELECT lower(substring(t.%(mail_field)s, '([^ ,;<@]+@[^> ,;]+)'))
               FROM mailing_trace s
```

Comment:
i would not include URL in commit message

Those line can be moved if we do a bugfix in it

Reply:
Hello, thanks for the review. 

I'm not sure to understand. Are you only talking about the commit message? If yes, indeed in this case, the url is useless. I generally use it with the hash reference in it. I will change that.

Reply:
yes about the commit :) thanks :)